src/Language/PureScript/Pretty/Types.hs

```diff
@@ -218,6 +219,19 @@ prettyPrintTypeWithUnicode = prettyPrintType' unicodeOptions
 prettyPrintSuggestedType :: Type -> String
 prettyPrintSuggestedType = prettyPrintType' suggestingOptions
 
+-- | Generate a pretty-printed string newtype
+prettyPrintNewtypeForType :: ProperName 'TypeName -> [(Text, Maybe Kind)] -> Type -> String
+prettyPrintNewtypeForType (ProperName name) args ty =
+  "newtype " ++ name' ++ args' ++ " = " ++ name' ++ " " ++ ctr
+  where
+    name' = T.unpack name
+    args' = case args of
+      [] -> ""
```

Comment:
In this section, I just want to print out an empty string if I don't have any type arguments to my type synonym. Should I rewrite this section in some way?